### DISCUSSION TITLE
Unsure about the behavior of useDialogStore

### DISCUSSION THREAD

**Discussion by crisshaker** [ariakit/ariakit#3123] [Questions]
I'm not sure if this is a bug or expected behavior.

Based on my understanding, when using the code snippet:
```tsx
const dialogStore = useDialogStore({ open: true });
```
the dialog should remain 'open' regardless of any actions like clicking outside or pressing the escape key. 

However, it seems that this behavior is currently broken as the dialog is closing. Please correct me if this is expected behavior.

**Comment by jasikpark** [comment]
I think I would expect that to configure the initial state of the dialog? 

https://ariakit.org/reference/use-dialog-store#open

I think if you really wanted to prevent closing it, you could manage that in https://ariakit.org/reference/use-dialog-store#setopen

**Comment by diegohaz** [comment]
That could be a bug. Could you please open a new issue?

**Reply by crisshaker** [reply]
Just did: https://github.com/ariakit/ariakit/issues/3125

**Reply by crisshaker** [reply]
@diegohaz There's a related bug I stated in the issue.
